src/less/wizard-stepper/base/wizard-stepper.less

```diff
@@ -0,0 +1,252 @@
+    background-color: @wizard-step-default-color !important;
+}
+
+.wizard-stepper__transition--upcoming .wizard-stepper__icon::after {
+    background-color: @wizard-step-upcoming-color !important;
+}
+
+.wizard-stepper__transition--inprogress .wizard-stepper__icon::after {
+    background-color: @wizard-step-confirmation-color !important;
+}
+
+.wizard-stepper__transition--confirmation .wizard-stepper__icon::after {
+    background-color: @wizard-step-confirmation-color !important;
+}
+
+.wizard-stepper__transition--insufficient .wizard-stepper__icon::after {
+    background-color: @wizard-step-insufficient-color !important;
+}
+
+.wizard-stepper__transition--error .wizard-stepper__icon::after {
+    background-color: @wizard-step-error-color !important;
+}
+
+.wizard-stepper__item {
+    min-width: 80px;
+    text-align: center;
+
+    .wizard-stepper__text {
+        margin-top: 8px;
+    }
+
+    &&--default {
+        color: @wizard-step-default-color;
+
+        .wizard-stepper__icon::before,
+        .wizard-stepper__icon::after {
+            background-color: @wizard-step-default-color;
+        }
+
+        .wizard-stepper__text {
+            color: @wizard-step-default-color;
+        }
+    }
+
+    &&--upcoming {
+        color: @wizard-step-upcoming-color;
+
+        .wizard-stepper__icon::before,
+        .wizard-stepper__icon::after {
+            background-color: @wizard-step-upcoming-color;
+        }
+    }
+
+    &&--confirmation .wizard-stepper__text.wizard-stepper__text--bold {
+        color: @wizard-step-default-color;
+        font-weight: bold;
+    }
+
+    &&--confirmation {
+        color: @wizard-step-title-light-color;
+    }
+
+    &&--confirmation,
+    &&--inprogress {
+        .wizard-stepper__icon::before,
+        .wizard-stepper__icon::after {
+            background-color: @wizard-step-confirmation-color;
+        }
+    }
+
+    &&--confirmation,
+    &&--upcoming {
+        .wizard-stepper__text {
+            color: @wizard-step-title-light-color;
+        }
+    }
+
+    &&--inprogress {
+        svg.icon {
+            color: @wizard-step-confirmation-color;
+        }
+
+        .wizard-stepper__text {
+            color: @wizard-step-default-color;
+            font-weight: bold;
+        }
+    }
+
+    &&--insufficient .wizard-stepper__icon::before {
+        background-color: @wizard-step-insufficient-color;
+    }
+
+    &&--error .wizard-stepper__icon::before {
+        background-color: @wizard-step-error-color;
+    }
+
+    &:nth-child(1) {
+        margin-left: 0;
+
+        .wizard-stepper__icon::before {
+            height: 0;
+        }
+    }
+
+    &:nth-last-child(1) {
+        margin-left: 0;
+
+        .wizard-stepper__icon::after {
+            height: 0;
+        }
+    }
+}
+
+.wizard-stepper--fluid {
+    & > .wizard-stepper__item {
+        flex: 1;
+        max-width: 220px;
+    }
+}
+
+.wizard-stepper.wizard-stepper--vertical {
```

Comment:
Do we need the extra specificity of a chained class here? Ideally would just be `.wizard-stepper--vertical`.